bin/run-linkchecker.sh

```diff
@@ -9,8 +9,8 @@ CMD="linkchecker"
 CMD="${CMD} -F html/utf8//results/linkchecker-out.html"
 if [ -n "${THREADS}" ]; then CMD="${CMD} -t ${THREADS}"; fi
 if [ -n "${RECURSION_LEVEL}" ]; then CMD="${CMD} -r ${RECURSION_LEVEL}"; fi
-if [ -n "${VERBOSE}" ]; then CMD="${CMD} -v"; fi
-if [ -n "${CHECK_EXTERNAL}" ]; then CMD="${CMD} --check-extern"; fi
+if [ "${VERBOSE}" = "true" ]; then CMD="${CMD} -v"; fi
+if [ "${CHECK_EXTERNAL}" = "true" ]; then CMD="${CMD} --check-extern"; fi
```

Comment:
Optionally you can lowercase the env variable to make `True` and `TRUE` work too.
`CHECK_EXTERNAL=`echo "$CHECK_EXTERNAL" | tr '[:upper:]' '[:lower:]'`